src/AppBundle/Entity/Location.php

```diff
@@ -216,4 +238,38 @@ public function getAddress()
     {
         return $this->address;
     }
+
+    /**
+     * Add voteSetting
+     *
+     * @param \AppBundle\Entity\VoteSettings $voteSetting
+     *
+     * @return Location
+     */
+    public function addVoteSetting(\AppBundle\Entity\VoteSettings $voteSetting)
+    {
+        $this->voteSetting[] = $voteSetting;
```

Comment:
I say you talk about diferent function, f you have some logic, aboyt uniq element collection, better create new function addUniqElement and in this function check uniw or not